main/pipeline/pipeline.cc

```diff
@@ -226,6 +226,9 @@ vector<ast::ParsedFile> incrementalResolve(core::GlobalState &gs, vector<ast::Pa
         if (opts.stripePackages) {
             Timer timeit(gs.tracer(), "incremental_packager");
             what = packager::Packager::runIncremental(gs, move(what));
+            if (opts.stopAfterPhase == options::Phase::PACKAGER) {
+                return what;
+            }
```

Comment:
In the interest of making this PR a little bit more digestible (since it's pretty huge) WDYT about taking the `pipeline.cc` changes (and some of the supporting option changes) and moving them to a separate PR?

Reply:
Can break this out. Tbh I needed it more for testing this feature, but it’s unrelated, and can be made into a separate feature.